src/runtime/native/TypeOffset.cs

```diff
@@ -88,7 +94,7 @@ internal static void Use(ITypeOffsets offsets)
                 int value = (int)sourceProperty.GetValue(offsets, null);
                 offsetProperty.SetValue(obj: null, value: value, index: null);
             }
-
+            Size = (int)Marshal.ReadIntPtr(Runtime.PyTypeType, tp_basicsize); ;
```

Comment:
NIT: Probably not an issue, but this assumes `Runtime` is already initialized, and the rest of the code in this file does not.
I'd rather you move this `Size` elsewhere, where it is known for sure `Runtime` is initialized.

Also: extra semicolon.

Reply:
`ABI.Initialize` called after all  Runtime.PyXXXX initialized, so it may not be a problem, if it does, bugs...